src/game.js

```diff
@@ -1,37 +1,60 @@
-const dbGame = []
 const idHelper = require('./IdHelper.js')
+const GameBd = require('.././database/GameModel.js')
+const BoardBd = require('.././database/BoardModel.js')
 
 class Game {
-	constructor({cols = 10, rows = 10} = {}){
-		this.cols = cols;
-		this.rows = rows;
-	}
-	static create({cols = 10, rows = 10} = {}) {
-		const game = new Game({cols,rows});
-		game.id = dbGame.length + 1;
-		game.playerId = idHelper();
-		const token = idHelper();
-		game.token = token
-		dbGame.push(game);
-		game.session = `http://localhost:3000/game?token=${token}`;
+  constructor({cols = 10, rows = 10} = {}) {
+    this.cols = cols;
+    this.rows = rows;
+  }
 
-		return Promise.resolve({
-			id : game.id, 
-			session : game.session,
-			playerId : game.playerId
-		})
-	}
+  static create({cols = 10, rows = 10} = {}) {
+    const game = new Game({cols,rows});
+    game.playerId = idHelper();
+    const token = idHelper();
+    game.token = token
+    game.session = `http://localhost:3000/game?token=${token}`;
+    return GameBd.sync()
+      .then(() => {
+        return GameBd.create({ token, playerId1 : game.playerId })
+      })
+      .then(() => {
+        return GameBd.findOne({ where: { token } })
```

Comment:
I think you don't need to find the game by its token, if the "create" works you just assume that everything went fine, besides that the "create" method returns the newly created object:
```javascript
return GameBd.sync()
       .then(() => {
         return GameBd.create({ token, playerId1 : game.playerId })
       })
       .then(gameModel =>  { . . . })
```

Reply:
It was applied in the last commit